adapters/src/main/java/io/realm/RealmRecyclerViewAdapter.java

```diff
@@ -72,12 +72,16 @@ public void onChange(Object collection, OrderedCollectionChangeSet changeSet) {
 
                 OrderedCollectionChangeSet.Range[] modifications = changeSet.getChangeRanges();
                 for (OrderedCollectionChangeSet.Range range : modifications) {
-                    notifyItemRangeChanged(range.startIndex, range.length);
+                    notifyItemRangeChanged(range.startIndex + dataOffset(), range.length);
                 }
             }
         };
     }
 
+    public int dataOffset() {
```

Comment:
Please add Javadoc explaining what this does.

Reply:
Just did